js/index.js

```diff
@@ -2,22 +2,26 @@
 
 function updateSubtotal(product) {
   console.log('Calculating subtotal, yey!');
-
+  let price = parseFloat(product.querySelector('.price span').innerHTML);
+  let quantity = parseInt(product.querySelector('.quantity input').value);
+  let subTotal = price * quantity;
+  console.log(subTotal);
+  product.querySelector('.subtotal span').innerHTML = subTotal;
+  return subTotal;
   //... your code goes here
 }
 
 function calculateAll() {
   // code in the following two lines is added just for testing purposes.
   // it runs when only iteration 1 is completed. at later point, it can be removed.
-  const singleProduct = document.querySelector('.product');
-  updateSubtotal(singleProduct);
-  // end of test
+  let total = 0;
+  const singleProduct = document.querySelectorAll('.product');
```

Comment:
Is the name "singleProduct" apt when multiple nodes with class "product" are selected?